v2/tests/log_sink_etw_int/log_sink_etw_int.c

```diff
@@ -0,0 +1,683 @@
+                metadata++;
+                break;
+            case TlgInANSISTRING:
+                (void)strncpy(parsed_events[current_parsed_event_count].properties[property_index].ascii_char_ptr_value, current_user_data_field, sizeof(parsed_events[current_parsed_event_count].properties[property_index].ascii_char_ptr_value));
+                parsed_events[current_parsed_event_count].properties[property_index].ascii_char_ptr_value[sizeof(parsed_events[current_parsed_event_count].properties[property_index].ascii_char_ptr_value) - 1] = '\0';
+                current_user_data_field += strlen(current_user_data_field) + 1;
+                break;
+            case TlgInUINT8:
+                parsed_events[current_parsed_event_count].properties[property_index].uint8_t_value = *(uint8_t*)current_user_data_field;
+                current_user_data_field += sizeof(uint8_t);
+                break;
+            case TlgInINT8:
+                parsed_events[current_parsed_event_count].properties[property_index].int8_t_value = *(int8_t*)current_user_data_field;
+                current_user_data_field += sizeof(int8_t);
+                break;
+            case TlgInUINT16:
+                parsed_events[current_parsed_event_count].properties[property_index].uint16_t_value = *(uint16_t*)current_user_data_field;
+                current_user_data_field += sizeof(uint16_t);
+                break;
+            case TlgInINT16:
+                parsed_events[current_parsed_event_count].properties[property_index].int16_t_value = *(int16_t*)current_user_data_field;
+                current_user_data_field += sizeof(int16_t);
+                break;
+            case TlgInUINT32:
+                parsed_events[current_parsed_event_count].properties[property_index].uint32_t_value = *(uint32_t*)current_user_data_field;
+                current_user_data_field += sizeof(uint32_t);
+                break;
+            case TlgInINT32:
+                parsed_events[current_parsed_event_count].properties[property_index].int32_t_value = *(int32_t*)current_user_data_field;
+                current_user_data_field += sizeof(int32_t);
+                break;
+            case TlgInUINT64:
+                parsed_events[current_parsed_event_count].properties[property_index].uint64_t_value = *(uint64_t*)current_user_data_field;
+                current_user_data_field += sizeof(uint64_t);
+                break;
+            case TlgInINT64:
+                parsed_events[current_parsed_event_count].properties[property_index].int64_t_value = *(int64_t*)current_user_data_field;
+                current_user_data_field += sizeof(int64_t);
+                break;
+            }
+
+            property_index++;
+        }
+
+        parsed_events[current_parsed_event_count].property_count = property_index;
+
+        (void)InterlockedIncrement(&parsed_event_count);
+        WakeByAddressAll((void*)&parsed_event_count);
+    }
+}
+
+static char test_path[MAX_PATH];
```

Comment:
>test_path [](http://example.com/codeflow?start=12&length=9)

Is thsi really used? Seems to be filled with some temp path later and then nothing happens to it.

Reply:
Leftover from the times of logging in a file.